### PR TITLE
5 get api reviews review

### PR DESCRIPTION
**Pull request by Deehjay**

### CODE REVIEW

File: __tests__/index.test.js
Comment (callumu21) [inline __tests__/index.test.js:79]: This test should be asserting that the review with id 1 is returned, not just any id. You also know what this should look like from the test database, so feel free to assert against the values you know should be returned.

File: controllers/reviews.controllers.js
Comment (callumu21) [inline controllers/reviews.controllers.js:15]: I'd recommend just returning on a key of 'review' for this one, but selectedReview is okay.

**Review by callumu21 — CHANGES_REQUESTED** [review]
Sound logic here, but be careful that tests are checking what they should be. If you could change the first test for this endpoint as per my comments and submit another PR, it should be good to merge after approval 👍

**Comment by cjplatten** [comment]
Changes look good!

**Pull request merged**
